app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,54 @@
+class TasksController < ApplicationController
+  def index
+    @list = Task.all
+
+
+  end
+
+  def new
+    @list = Task.new
+  end
+
+  def show
+    @list = Task.find(params[:id])
+  end
+
+  def create
+    @list = Task.new(task_params)
+    @list.save
+    if @list.save
```

Comment:
You really only need to call save once - in this case it would probably make best sense to call in the `if` as opposed to outside it